Replicator/Replicator.cc

```diff
@@ -48,7 +50,8 @@ namespace litecore { namespace repl {
     static constexpr StoppingErrorEntry StoppingErrors[] = {
         {{ LiteCoreDomain, kC4ErrorUnexpectedError,0 }, true, "An exception was thrown"_sl},
         {{ WebSocketDomain, 403, 0}, true, "An attempt was made to perform an unauthorized action"_sl},
-        {{ WebSocketDomain, 503, 0 }, false, "The server is over capacity"_sl}
+        {{ WebSocketDomain, 503, 0 }, false, "The server is over capacity"_sl},
+        {{ LiteCoreDomain, kC4ErrorRemoteError, 1 }, true, "Unexpected error from remote"_sl }
```

Comment:
Why is this 1 here instead of 0 for the third part of of C4Error?

Reply:
Now that you point it out, I see that it's probably unnecessary here, as it's not compared. But that's what the error from the SG response had. (`internal_info == 1`)